_sass/custom/custom.scss

```diff
@@ -73,3 +74,44 @@ p, ul, ol {
     margin: 1.7rem 0 1rem 0;
     font-size: 1.6rem;
 }
+
+.custom-card-wrapper {
+    a {
+      color: $gray-800;
+  
+      &:hover {
```

Comment:
Add a light gray background color for the card background upon hover. We can use Bootstrap's $gray-100 color variable for now.